doc/user-manual/language/copatterns.lagda.rst

```diff
@@ -21,7 +21,9 @@
 Copatterns
 **********
 
-Consider the following record:
+If you are looking for information on how to use copatterns with 
```

Comment:
I am not sure if this is the correct place to reference the coinduction section, but it didn't feel like it could go anywhere else naturally either.